staging-antora-playbook.yml

```
@@ -81,7 +84,7 @@ asciidoc:
   - ./lib/tabs-block.js
 ui:
   bundle:
-    url: https://github.com/couchbase/docs-ui/releases/download/v201/ui-bundle.zip
+    url: https://github.com/couchbase/docs-ui/releases/download/v202/ui-bundle.zip
```

```suggestion
    url: https://github.com/couchbase/docs-ui/releases/download/v201/ui-bundle.zip
```

I suggest bumping the version as part of https://github.com/couchbase/docs-site/pull/61. Otherwise the Scala item on the top nav will 404.